Playmate/View Controllers/Profile Tab/ProfileViewController.m

```diff
@@ -53,24 +52,34 @@ - (void)viewWillAppear:(BOOL)animated {
         [self.profileImageView setImage:img];
         self.profileImagePlaceholder.alpha = 0;
     }
-    if (user[@"backdropImage"] != nil) {
-        // set image stuff
-        UIImage* img = [UIImage imageWithData:[user[@"backdropImage"] getData]];
-        [self.backdropImageView setImage:img];
-    }
-    else {
-        UIImage* img = [UIImage imageNamed:@"playmate_backdrop.png"];
-        [self.backdropImageView setImage:img];
-    }
```

Comment:
why we do not need this anymore?

Reply:
I decided to put off the backdrop image feature for later since it isn't really necessary, but I might consider adding it back when I specifically work on polishing the UI of my app.